crates/sui-cluster-test/src/lib.rs

```diff
@@ -0,0 +1,138 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+use async_trait::async_trait;
+use clap::*;
+use cluster::{Cluster, ClusterFactory};
+use config::{ClusterTestOpt, Env};
+use std::sync::Arc;
+use sui::client_commands::WalletContext;
+use sui_core::gateway_state::GatewayClient;
+
+use crate::faucet::{Faucet, FaucetFactory};
+use sui_json_rpc_types::TransactionResponse;
+use sui_types::gas_coin::GasCoin;
+use sui_types::{
+    base_types::SuiAddress,
+    messages::{Transaction, TransactionData},
+};
+use tokio::time::{sleep, Duration};
+use tracing::{error, info};
+use wallet_client::WalletClient;
+
+pub mod cluster;
+pub mod config;
+pub mod faucet;
+pub mod helper;
+pub mod test_case;
+pub mod wallet_client;
+
+#[allow(unused)]
+pub struct TestContext {
+    /// The cluster env
+    env: Env,
```

Comment:
this shouldn't be here, we don't want tests to care about the env they are running in